lexers/j/jsgf.go

```diff
@@ -0,0 +1,19 @@
+package j
+
+import (
+	. "github.com/alecthomas/chroma" // nolint
+	"github.com/alecthomas/chroma/lexers/internal"
+)
+
+// Jsgf lexer.
+var Jsgf = internal.Register(MustNewLexer(
+	&Config{
+		Name:      "JSGF",
+		Aliases:   []string{"jsgf"},
+		Filenames: []string{"*.Jsgf"},
```

Comment:
typo: should be lower case
```suggestion
		Filenames: []string{"*.jsgf"},
```

Reply:
tks.